tests/test_full_model.py

```diff
@@ -33,8 +34,15 @@ def test_full_model_exact_match(atol_forward=1e-3, atol_inference=1e-3):
         embs = model.transformer.word_embeddings_layernorm(embs)
         recurrent_outputs = []
         with model.transformer.h.inference_session(max_length=embs.shape[1]) as sess:
+            if pass_empty_tensors:
+                recurrent_outputs.append(sess.step(torch.empty(1, 0, config.hidden_size)))
+
             for t in range(embs.shape[1]):
                 recurrent_outputs.append(sess.step(embs[:, t : t + 1, :]))
+                if t == 5 and pass_empty_tensors:
```

Comment:
Better to replace a hardcoded constant by something like `test_inputs.size(1)`

Reply:
it was an arbitrary constant, but i've made it a function of length